deploy/chart/templates/20-packageserver.yaml

```diff
@@ -0,0 +1,146 @@
+apiVersion: apiregistration.k8s.io/v1beta1
+kind: APIService
+metadata:
+  name: v1alpha1.apps.coreos.com
+spec:
+  insecureSkipTLSVerify: true
+  group: apps.coreos.com
```

Comment:
for actually deploying we probably want to think about this group name

Reply:
I was thinking something like `packages.apps.redhat.com`